runner/src/run.rs

```diff
@@ -24,6 +24,17 @@ const QEMU_ARGS: &[&str] = &[
     "-machine", "virt",
 ];
 
+const UBUNTU_ARGS: &[&str] = &[
+    "-device",
+    "virtio-net-device,netdev=eth0",
+    "-netdev",
+    "user,id=eth0",
+    "-device",
+    "virtio-rng-pci",
+    "-drive",
+    "file=misc/ubuntu-miralis.img,format=raw,if=virtio",
```

Comment:
Where does this file come from? Is there anything special that we need to do to generate it or is it simply a disk image you downloaded somewhere?

Reply:
A simple disk image